charts/kong/example-values/minimal-kong-standalone.yaml

```diff
@@ -0,0 +1,19 @@
+# The minimum values.yaml configuration necessary for Kong without the ingress controller, using the Postgres subchart
+image:
+  repository: kong
+  tag: "2.0"
```

Comment:
Why do you need to set these if this is "minimal"? These are defaults, right?

Reply:
Consistency, for the most part--different images aren't something we're able to represent in the main values.yaml, so that's a major facet of what these examples set.

These aren't really intended to be "the absolute minimum diff from the base values.yaml" but more just minimalist configurations that don't set up much. Can rename them to "basic" instead if that makes more sense.

Reply:
Re-worded comments, filenames left as-is: I forget if renaming them makes it harder to read the diff. Please review the remainder, and if that looks good, I'll rename them "basic" instead of "minimal".